### PR TITLE
Use safe mode by default in clvm. Only allow defined opcodes.

### PR DESCRIPTION
**Pull request by aqk**

### CODE REVIEW

**Comment by aqk** [comment]
This patch makes us always use strict mode, as intended during blockchain validation.

File: chia/util/condition_tools.py [hunk 1]
```diff
@@ -159,6 +159,7 @@ def conditions_for_solution(
 ) -> Tuple[Optional[Err], Optional[List[ConditionWithArgs]], uint64]:
     # get the standard script for a puzzle hash and feed in the solution
     try:
+        # TODO: Note this call to Program.run_with_cost
```
Comment (aqk) [inline chia/util/condition_tools.py:162]: Note that we call `Program.run_with_cost`, which is `chia.types.program.run_program` in chia/util/condition_tools.py

**Comment by arvidn** [comment]
my understanding is that the intention is to only use strict mode when accepting transactions into our mempool. If some other farmer is signing a block that validates despite having unknown operators in it, we should accept that.

**Comment by aqk** [comment]
@arvidn Thanks. I'll hold off on this.

**Pull request closed without merging**